src/opentslm_sp.py

```
@@ -36,30 +34,30 @@ def __init__(self, llm_id: str = "mlx-community/Llama-3.2-1B-Instruct-4bit"):
 
-        projector_weights = _convert_projector_weights(checkpoint["projector_state"])
-        self.projector.load_weights(list(projector_weights.items()))
+        self.projector.load_weights(list(projector_state.items()))
 
-        if checkpoint.get("lora_state"):
-            self._apply_lora(checkpoint["lora_state"])
+        try:
+            lora_state = mx.load(f"{path}.lora.safetensors")
+            self._apply_lora(lora_state)
+        except Exception:
+            pass
```

Ruff (0.15.9)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)